ci/build.yml

```diff
@@ -6,7 +6,7 @@ image_resource:
   type: docker-image
   source:
     repository: golang
-    tag: 1.15
+    tag: latest
```

Comment:
We need to specify a specifc version of 'go' for traceability and tracking down bugs, please revert this.

Reply:
As per discussion - this is a separate pipeline for libraries, so taking the latest version is preferred as an early warning to potential issues up ahead.